srml/council/src/lib.rs

```diff
@@ -14,7 +14,221 @@
+//! Each proposal has a minimum threshold of yay votes that it needs to gain to be enacted.
+//!
+//! - **Council motion:** A mechanism used to enact a proposal.
+//! - **Proposal:** A submission by a councillor. An initial vote of yay from that councillor is applied.
+//! - **Vote:** A vote of yay or nay from a councillor on a single proposal. Councillors may change their vote but a
+//!   duplicate vote will return an error.
+//!
+//! Upon each vote, if the threshold is reached, the proposal is dispatched from the `Council` origin. Similarly,
+//! if the number of nay votes is high enough such that it could not pass even if all other councillors
+//! (including those who have not voted) voted yay, the proposal is dropped.
+//!
+//! Note that a council motion has a special origin type, [`seats::Origin`](./motions/enum.Origin.html), that limits
+//! which calls can be effectively dispatched. The [Treasury](../srml_treasury/index.html) module is an example
+//! that uses the council motion origin.
+//!
+//! #### Council Voting (voting.rs)
+//!
+//! Proposals that are proposed by and voted upon by councillors. Unlike motion proposals, if a proposal is approved,
+//! it is _elevated_ to the [`Democracy Module`](../srml_democracy/index.html) module as a referendum.
+//!
+//! - **Proposal validity:** A council proposal is valid when it's unique, hasn't yet been vetoed, and
+//! when the proposing councillor's term doesn't expire before the block number when the proposal's voting period ends.
+//! A proposal is a generic type that can be _dispatched_ (similar to variants of the `Call` enum in each module).
+//! - **Proposal postponement:** Councillors may postpone a council proposal from being approved or rejected.
+//! Postponement is equivalent to a veto, which only lasts for the cooloff period.
+//! - **Cooloff period:** Period, in blocks, for which a veto is in effect.
+//! - **Referendum:** The means of public voting on a proposal.
+//! - **Veto:** A council member may veto any council proposal that exists. A vetoed proposal that's valid is set
+//! aside for a cooloff period. The vetoer cannot re-veto or propose the proposal again until the veto expires.
+//! - **Elevation:** A referendum can be _elevated_ from the council to a public referendum. This means it has
+//! been passed to the Democracy module for a public vote.
+//! - **Referendum cancellation:** At the end of a given block we cancel all elevated referenda whose voting period
+//! ends at that block and where the outcome of the vote tally was a unanimous vote to cancel the referendum.
+//! - **Voting process to elevate a proposal:** At the end of a given block we tally votes for expiring referenda.
+//! Referenda that are passed (yay votes are greater than nay votes plus abstainers) are sent to the Democracy
+//! module for a public referendum. If there are no nay votes (abstention is acceptable), then the proposal is
```

Comment:
remove `module` from start of this like if the suggestion to add a link to Democracy Module is accepted on the previous line.
```suggestion
//! for a public referendum. If there are no nay votes (abstention is acceptable), then the proposal is
```